# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/moz.build
# Commit: 287bedb227e8
# Full Hash: 287bedb227e89933347633242efe1acd4236ad70
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-12 09:32:07
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r a01c165a2524 -r 287bedb227e8 dom/webauthn/moz.build
--- a/dom/webauthn/moz.build	Wed Feb 12 02:12:58 2025 +0000
+++ b/dom/webauthn/moz.build	Wed Feb 12 02:12:58 2025 +0000
@@ -28,7 +28,7 @@
     "AuthenticatorAttestationResponse.h",
     "AuthenticatorResponse.h",
     "PublicKeyCredential.h",
-    "WebAuthnManager.h",
+    "WebAuthnHandler.h",
     "WebAuthnPromiseHolder.h",
     "WebAuthnTransactionChild.h",
     "WebAuthnTransactionParent.h",
@@ -43,7 +43,7 @@
     "PublicKeyCredential.cpp",
     "WebAuthnArgs.cpp",
     "WebAuthnAutoFillEntry.cpp",
-    "WebAuthnManager.cpp",
+    "WebAuthnHandler.cpp",
     "WebAuthnPromiseHolder.cpp",
     "WebAuthnResult.cpp",
     "WebAuthnService.cpp",
